glib-macros/src/gflags_attribute.rs

```diff
@@ -152,8 +152,7 @@ pub fn impl_gflags(input: &DeriveInput, gtype_name: &LitStr) -> TokenStream {
         impl #crate_ident::value::SetValue for #name {
             unsafe fn set_value(value: &mut #crate_ident::Value, this: &Self) {
                 #crate_ident::gobject_ffi::g_value_set_flags(
-                    #crate_ident::translate::ToGlibPtrMut::to_glib_none_mut(value).0,
-                    #crate_ident::translate::ToGlib::to_glib(this))
```

Comment:
This shouldn't be necessary. The type should be `Copy` so you can use `to_glib()` here as before.